src/templates/example.html

```diff
@@ -34,6 +34,14 @@
         </div>
       </div>
       {{/sections}}
+      <div style="text-align: center">
+        <div style="display: inline-block; margin: 0 auto;">
+          <a href="{{prev}}">Prev</a>
+        </div>
+        <div style="display: inline-block; margin: 0 auto;">
+          <a href="{{next}}">Next</a>
```

Comment:
Two things:

1. Can we include the title of the next sample?
2. Make the next element optional. You can define predicates with mustache:

```
{{#next}}
<div style="display: inline-block; margin: 0 auto;">
   <a href="{{next}}">Next: {{nextTitle}}</a>
</div>
{{/next}}
```